apps/marketplace/src/temp/data.ts

```diff
@@ -51,15 +51,15 @@ export const mockSales: ReadonlyArray<Sale> = [
     costAssetName: "XYZ123",
     costPolicyId: "123XYZ",
     createdAt: new Date("April 10th, 2024").toDateString(),
-    id: "3cfb2d02-a320-4385-96d1-1498d8a1df58",
+    id: "cec4f704-69c4-41fd-807d-47aa2d73f0d2",
```

Comment:
Added this just for testing, this is one of the backend IDs, so clicking the first track under `Just Released` leads to a backend example. The First two play icons under `Just Released` can playback from a `nftcdn` source that might be similar to what the end url will look like.

Didn't want to change this file up too much since we might be pulling in backend data soon and replacing all of this.